modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/Field/Field.es.js

```diff
@@ -155,6 +157,7 @@ const FieldLazy = ({
 			visible
 			{...field}
 			{...otherProps}
+			{...module.properties}
```

Comment:
From @brunobasto at https://github.com/liferay-frontend/liferay-portal/pull/196#discussion_r463880901

> Instead of having this new properties can't we just add editorConfig directly in the JSON?

From @matuzalemsteles at https://github.com/liferay-frontend/liferay-portal/pull/196#discussion_r463883717

> Yes, I was thinking of just reducing the hack a little so it doesn't look so exclusive to the editorConfig 🤪

Calling it `properties` is a bit confusing... maybe call it `module.configuration` or something like that? All fields could have configuration and we could expand on that in the future for this field in particular so toolbars can be set per-field rather than globally.

Reply:
we can switch to `configuration`. I'm not sure if we want this pattern haha ​​😅, this was the only place that I ended up seeing with Bruno to put this information on because we couldn't add it directly to the fields, because the settings were just passed in the Form view mode and we need it in Builder too. But maybe it could be a pattern because there seems to be a case!